### PR TITLE
Fix for #3. UI Tests don't work on OSX

### PR DESCRIPTION
**Pull request by cdietrich**

### CODE REVIEW

File: jbase.releng/pom.xml [hunk 1]
```diff
@@ -358,38 +358,7 @@
 						</artifact>
 					</target>
 					<environments>
-						<environment>
```
Comment (LorenzoBettini) [inline jbase.releng/pom.xml]: is it really required to remove the environment specifications to make it work?

File: domainmodel/org.eclipse.xtext.example.domainmodel.tests/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -18,6 +18,7 @@ Require-Bundle: org.eclipse.xtext.example.domainmodel,
  org.eclipse.xtext.common.types.ui,
  org.eclipse.jdt.core;bundle-version="3.6.0",
  org.objectweb.asm;bundle-version="[5.0.1,6.0.0)";resolution:=optional,
+ org.eclipse.jdt.launching.macosx;resolution:=optional,
```
Comment (LorenzoBettini) [inline domainmodel/org.eclipse.xtext.example.domainmodel.tests/META-INF/MANIFEST.MF]: can it be without .macosx?

File: jbase.tests/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -11,6 +11,7 @@ Require-Bundle: jbase,
  org.eclipse.xtext.junit4,
  org.eclipse.xtext.xbase.lib,
  org.eclipse.ui.workbench;resolution:=optional,
+ org.eclipse.jdt.launching;resolution:=optional,
```
Comment (LorenzoBettini) [inline jbase.tests/META-INF/MANIFEST.MF]: here you don't specify .macosx, so jdt.launching is enough?

File: .travis.yml [hunk 4]
```diff
@@ -2,6 +2,10 @@ sudo: false
 
 language: java
 
+os:
```
Comment (LorenzoBettini) [inline .travis.yml]: Please make sure you're on top of my master branch, where I reverted this experiment which makes the travis job fail anyway

**Comment by LorenzoBettini** [comment]
@cdietrich have you closed this intentionally?

**Comment by cdietrich** [comment]
yes i dont get this github workflow done. i did a new pullreq

**Comment by LorenzoBettini** [comment]
OK, I'm having a look at the new pull request

**Pull request closed without merging**